src/js/components/molecules/main-nav-item/main-nav-item.ts

```diff
@@ -0,0 +1,124 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import { NavigationItem  } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/navigation-item/NavigationItem';
+import BrowserCapabilities from 'utils/browser-capabilities';
+
+@Component({
+    name: 'main-nav-item',
+    template: require('./main-nav-item.html'),
+})
+
+export default class MainNavItem extends Vue  {
+    @Prop({ type: Boolean, required: false }) mainNavOpen?: boolean;
+
+    @Prop({ type: String, required: true }) label!: string;
+    @Prop({ type: String, required: false }) url?: string;
+    @Prop({ type: Array, required: false }) items?: NavigationItem[];
+
+    @Prop({ type: Object, required: false }) bottomLink?: object;
+    @Prop({ type: String, required: true }) itemId!: string;
+    @Prop({ type: Boolean, required: false }) isCompact?: boolean;
+    @Prop({ type: Boolean, required: true }) active!: boolean;
+
+    flyoutIsOpen: boolean =  false;
+    hideTimeout: null = null;
+    isTouch: boolean = BrowserCapabilities.supportsTouch;
+
+    get isActive(): string | boolean {
```

Comment:
Doesn't make sense to have a isActive which returns 'page', is there better solution?